modules/DMZ/data_kit/dataset_insight.py

```diff
@@ -1,7 +1,9 @@
 # This is for analyzing columns or datasets, to figure out how to best act upon them.
 import sys
 import os
+
 sys.path.insert(0, os.path.abspath('../../..'))
```

Comment:
Team: There seriously has to be a better way to include things that using this "hack"